controller_interface/src/controller_interface.cpp

```diff
@@ -73,9 +74,20 @@ return_type ControllerInterface::init(const std::string & controller_name)
 
 const rclcpp_lifecycle::State & ControllerInterface::configure()
 {
-  update_rate_ = node_->get_parameter("update_rate").as_int();
+  // TODO(destogl): this should actually happen in "on_configure" but I am not sure how to get
```

Comment:
@bmagyar please comment on this. Maybe you have an idea how to achieve the behavior I described.

Reply:
This is also a valid approach, for sure. @bmagyar what do you think?

The only “drawback” from my side is that uses have to explicit call this on the parent class. If someone forgets to do this, it may come with an issue that this is not working.

Reply:
@bmagyar @destogl We could then maybe refactor parts in the [`hardware_interface`](https://github.com/ros-controls/ros2_control_demos/blob/96ef3b5e28802189aa0c7ac54c36098aa581d037/ros2_control_demo_hardware/src/rrbot_system_position_only.cpp#L31-L34)